tests/test_cli.py

```diff
@@ -32,16 +32,17 @@ def _get_subprocess_env():
 
 
 SUBPROCESS_ENV = _get_subprocess_env()
+call = partial(subprocess.call, env=SUBPROCESS_ENV, shell=True)
 check_call = partial(subprocess.check_call, env=SUBPROCESS_ENV)
 check_output = partial(subprocess.check_output,  env=SUBPROCESS_ENV)
 
 xfail_if_no_git = pytest.mark.xfail(
-  subprocess.call(["git", "version"]) != 0,
```

Comment:
This looks incorrect. I think it either needs to be a single string or a list with multiple elements. I suspect that if you do that, you don't need `shell=True`.

Reply:
Thanks, you're right and a closer look at the Appveyor builds (https://ci.appveyor.com/project/c4urself/bump2version/builds/28485363/job/fhj33v7vcafmscr9) shows they actually gave a false positive and passed! There is a `strict` parameter than can be used to fail on XPASS, but that's another option for another day: http://doc.pytest.org/en/latest/skipping.html#strict-parameter

```
XPASS tests/test_cli.py::test_no_list_no_stdout[git] git is not installed
XPASS tests/test_cli.py::test_no_list_no_stdout[hg] hg is not installed
XPASS tests/test_cli.py::test_regression_tag_name_with_hyphens[git] git is not installed
XPASS tests/test_cli.py::test_unclean_repo_exception[git] git is not installed
============== 94 passed, 1 xfailed, 54 xpassed in 60.53 seconds ==============
___________________________________ summary ___________________________________
  py27: commands succeeded
  congratulations :)
Build success
```

Tested the single string method on Linux (Manjaro) and Windows locally, pushing the fix now :grinning:

Reply:
I still strongly prefer the list way because you don't have to worry about escaping, even though there are no variables in this case. If possible I also always avoid `shell=True` unless it's really needed.